### DISCUSSION TITLE
MASKs input Text_Field

### DISCUSSION THREAD

**Discussion by celsoabreu** [flet-dev/flet#4432] [Ideas]
### Discussed in https://github.com/flet-dev/flet/discussions/3454

<div type='discussions-op-text'>

<sup>Originally posted by **celsoabreu** June 12, 2024</sup>
Hello,
I'm new to this amazing framework. And I'm thankful to be part of this family.
Now, I see that the documentation is old(especially the tutorial section where people like me start their UI design journey).

issues:

-  Migrating a Flask solution developed in Python using Javascript, html, css as a front using geoprocessing, I verified that FLET did not miss the possibility of incorporating javascript tags like flas using the <scripts></scripts> tag, it would be good to miss this condition!
- Or you can use some libs to capture the user's geographic coordinates as we do with javascripts: navigator.geolocation.getCurrentPosition( 
                    function(position){
                        lat = position.coords.latitude;
                        lng = position.coords.longitude;
                    }
)
- Being able to format entries with TEXT_FIELD as we do in Python using MASK, or Jquery, create an example in the updated tutorial! 
- Implement an example with plotly using callback to create interactive and reactive graphs in real time, make it available in the documentation and in pip install plet --upgrade
- As the execution pattern is similar to FLASK, introduce the ORM, TEMPLATES and ROUTES functionalities of FLASK into FLET, this way we will have an absurd growth in migrations, enhancing the fusion with other frameworks

My Proposal is to make the tutorial more documented and explain the components as we go along with the tutorial and introduce new components.

I'm making these improvement suggestions because I'm sure they will add a lot of value to this fantastic framework that only tends to grow in our echo system. An amazing proposal that covers everything we needed in Python to go global once and for all, congratulations!

Very good framework, i not lock me use outher tecnology !!!!  
### THIS POST NEEDS CORE DEVELOPER'S ATTENTION

</div>

**Comment by celsoabreu** [comment]
wait returns!

**Comment by celsoabreu** [comment]
> wait returns!

Celso Abreu

**Comment by ndonkoHenri** [comment]
Taking note, thanks.
